azuredevops/internal/service/core/resource_team.go

```diff
@@ -0,0 +1,558 @@
+package core
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/ahmetb/go-linq"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+	"github.com/microsoft/azure-devops-go-api/azuredevops/core"
+	"github.com/microsoft/azure-devops-go-api/azuredevops/identity"
+	"github.com/microsoft/terraform-provider-azuredevops/azuredevops/internal/client"
+	securityhelper "github.com/microsoft/terraform-provider-azuredevops/azuredevops/internal/service/permissions/utils"
+	"github.com/microsoft/terraform-provider-azuredevops/azuredevops/internal/utils"
+	"github.com/microsoft/terraform-provider-azuredevops/azuredevops/internal/utils/converter"
+	"github.com/microsoft/terraform-provider-azuredevops/azuredevops/internal/utils/tfhelper"
+)
+
+func ResourceTeam() *schema.Resource {
+	return &schema.Resource{
+		Create:   resourceTeamCreate,
+		Read:     resourceTeamRead,
+		Update:   resourceTeamUpdate,
+		Delete:   resourceTeamDelete,
+		Importer: tfhelper.ImportProjectQualifiedResourceUUID(),
+		Schema: map[string]*schema.Schema{
+			"project_id": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validation.IsUUID,
+			},
+			"name": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validation.StringIsNotWhiteSpace,
+			},
+			"description": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"administrators": {
```

Comment:
Then you can't manage team administrators with the `azuredevops_team_administrators` resource.  Furthermore, I think that we should not always slavishly follow the functions in the web interface, since the group of users of the provider is different from the people who click around in the web page. We should make sure that Azure DevOps can be configured very flexible with the provider, even if that may mean breaking something, like having no administrator on a created team. This can be fixed easily.